packages/hash/frontend/src/pages/[account-slug]/types/entity-type/property-list-card.tsx

```diff
@@ -182,57 +186,204 @@ export const PropertyTypeRow = ({
+              [`.${tableRowClasses.root}:not(:hover) & .${outlinedInputClasses.root}:not(:focus-within)`]:
+                {
+                  boxShadow: "none",
+                  [`.${outlinedInputClasses.notchedOutline}`]: {
+                    borderColor: "transparent",
+                  },
+                  [`.${outlinedInputClasses.input}::placeholder`]: {
+                    color: "transparent",
+                  },
                 },
-              },
-          }}
-          inputProps={{ sx: { textOverflow: "ellipsis" } }}
-        />
-      </TableCell>
-      <TableCell>
-        <PropertyMenu onRemove={onRemove} property={property} />
-      </TableCell>
-    </TableRow>
-  );
+            }}
+            inputProps={{ sx: { textOverflow: "ellipsis" } }}
+          />
+        </CenteredTableCell>
+        <TableCell>
+          <PropertyMenu onRemove={onRemove} property={property} />
+        </TableCell>
+      </TableRow>
+      <Popper
+        open={multipleValuesMenuOpen}
+        anchorEl={anchorEl}
+        container={anchorEl}
+        placement="bottom"
+        sx={{ width: 1, zIndex: 1 }}
+        transition
+      >
+        {({ TransitionProps }) => {
+          return (
+            <ClickAwayListener
+              onClickAway={() => setMultipleValuesMenuOpen(false)}
+            >
+              <Fade {...TransitionProps}>
+                <Box
+                  sx={({ palette }) => ({
+                    border: 1,
+                    p: 1.5,
+                    background: palette.white,
+                    borderColor: palette.gray[30],
+                  })}
+                >
+                  <TextField
+                    type="number"
+                    size="small"
+                    label="Minimum"
+                    value={minValue ?? 0}
+                    onChange={(event) => {
+                      const value = parseInt(event.target.value, 10);
+                      if (value >= 0) {
+                        update(propertyIndex, {
+                          ...draftProperty,
+                          minValue: value,
+                          maxValue:
+                            !maxValue || value > maxValue ? value : maxValue,
+                        });
+                      }
+                    }}
+                    sx={{ mb: 2 }}
+                  />
+                  <TextField
+                    type="number"
+                    label="Maximum"
+                    value={maxValue ?? 0}
+                    onChange={(event) => {
+                      const value = parseInt(event.target.value, 10);
+                      if (value >= 0) {
+                        update(propertyIndex, {
+                          ...draftProperty,
+                          maxValue: value,
+                          minValue:
+                            !minValue || value < minValue ? value : minValue,
+                        });
+                      }
```

Comment:
As above, you should use `register`.

Reply:
Done in 669f2a5700d2e0a6385302b661eb077a24186144. As in the above comment, kept `minValue` out of the deps array.